go/src/koding/klientctl/commands/auth/register.go

```diff
@@ -50,8 +61,251 @@ func NewRegisterCommand(c *cli.CLI, aliasPath ...string) *cobra.Command {
 	return cmd
 }
 
+const (
+	trueValue  = "true"
+	falseValue = "false"
+)
+
+// RegisterRequest holds the registration request to Koding.
+type RegisterRequest struct {
+	Username        string `json:"username"`
+	FirstName       string `json:"firstName"`
+	LastName        string `json:"lastName"`
+	Password        string `json:"password"`
+	PasswordConfirm string `json:"passwordConfirm"`
+	Email           string `json:"email"`
+	Slug            string `json:"slug"`
+	CompanyName     string `json:"companyName"`
+	Newsletter      string `json:"newsletter,string"`
+	AlreadyMember   string `json:"alreadyMember,string"`
+	Agree           string `json:"agree"`
+}
+
 func registerCommand(c *cli.CLI, opts *registerOptions) cli.CobraFuncE {
```

Comment:
Could we for now disable register command (do not register it in main) until it's fixed?

#11027